.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: python
+python:
+  - "2.7"
+  - "3.5"
+  - "3.6"
+  - "3.7"
+  - "3.8"
+before_install:
+  - pip install poetry
+install:
+  - pip install tox-travis
+script:
+  - tox
```

Comment:
A bare `tox` runs `[tox] envlist` unless I'm mistaken and I don't think you want each of the pythons above running `py27,py35,py36,py37,py38`. I think you may need to spell things out a bit more per shard.